src/CTA.Rules.Models/ProjectActions.cs

```diff
@@ -14,6 +14,7 @@ public ProjectActions()
             FileActions = new BlockingCollection<FileActions>();
             PackageActions = new BlockingCollection<PackageAction>();
             ProjectReferenceActions = new BlockingCollection<string>();
+            ProjectLevelActions = new List<ProjectLevelAction>();
```

Comment:
Do we need this?

Reply:
I was getting null reference errors for it. Not sure about other collections, but List properties aren't initialized or given a default value unless specified.